tmessage/cli.py

```diff
@@ -19,9 +19,13 @@
 parser.add_argument('--user', action='store', type=str, required=True)
 parser.add_argument('--server', action='store', type=str)
 parser.add_argument('--port', action='store', type=int)
+parser.add_argument('--store',
+                    action='store_true',
+                    help='Store messages in JSON format.')

```

Comment:
Keep this in a single line.

Reply:
Also, make changes in the README as per the changes proposed in this PR.

Reply:
The documentation in README has been added for the new `--dont-store` argument